editor/import/editor_import_collada.cpp

```diff
@@ -89,10 +89,10 @@ struct ColladaImport {
 	Error _populate_skeleton(Skeleton *p_skeleton, Collada::Node *p_node, int &r_bone, int p_parent);
 	Error _create_scene_skeletons(Collada::Node *p_node);
 	Error _create_scene(Collada::Node *p_node, Spatial *p_parent);
-	Error _create_resources(Collada::Node *p_node, bool p_use_compression);
```

Comment:
This huge amount of modifications to the importers are unnecessary, Simply work with `ResourceImporterScene::_fix_node()` and add a `-softbody` import flag. When this flag is found, you convert the mesh to a softbody friendly format.

Reply:
In fact, you can just convert it to a SoftBody node there.

Reply:
Ok